site/content/docs/03-run-time.md

```diff
@@ -1081,3 +1082,22 @@ const { head, html, css } = App.render({
 	answer: 42
 });
 ```
+
+You can call the `.render()` with the following parameters:
+
+| option | default | description |
+| --- | --- | --- |
+| `props` | `{}` | An object of properties to supply to the component
+| `slots` | `{}` | A object of slots to supply to the component
```

Comment:
Is `slots` an incorrect copy-paste from other docs you've written which included slots handling?

I'm actually not sure what we want the API to look like here. It would have been nice if the first argument to `.render()` were an _object_ containing a `props` key, so that we could just add stuff alongside it. But, since that ship has already sailed, I think the next best thing would be to make the second argument be an object with `context` as a key (and, later, also add `slots` or whatever other options we wanted). We should try to avoid a bunch of positional parameters.

Reply:
yea so, i think we can discuss on this.

apparently right now the `.render()` function takes in 2 parameters, `.render(props, slots)`, though it is not documented anywhere.

definitely i would prefer passing `context` and `slot` as objects as the 2nd argument, if we are okay to change the undocumented 2nd argument.

Reply:
Oh I didn't even know that existed. I would say that's definitely a private API, like the `$$slots` option you can pass to client-side components. We should probably make it less discoverable anyway. I don't have a problem with making a change to that.